content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,199 @@
-| ------------- | ------------------------------------------------------------------------------------------------------------------------ | ------------------------- |
-| `GET /`       | Status                                                                                                                   | `handlers/status.js`      |
-| `GET /seed`   | Delete all database records and seed the database with test `users`, `profiles`, and `posts`. Returns the created users. | `handlers/seed.js`        |
-| `GET /users`  | Fetch all `users` in the database with their related `profiles`                                                          | `handlers/users.js`       |
-| `POST /users` | Create a single `users` in the database                                                                                  | `handlers/create-user.js` |
-| `GET /posts`  | Fetch all `posts` and their related `authors`                                                                            | `handlers/posts.js`       |
-
-To call the API, you can use curl. For example, the following command calls the `seed` endpoint. Replace `UNIQUE_IDENTIFIER` with the corresponding part of the URL from the output of `serverless deploy` in step 6.
+Now redeploy your application. To do so, run `serverless deploy`. The deployment output shows you that the correct Prisma engine binary is packaged and distributed, as follows:
 
-```terminal
-curl -v https://UNIQUE_IDENTIFIER.execute-api.us-east-1.amazonaws.com/dev/seed
+```code no-copy
+Serverless: Copy prisma schema for app...
+Serverless: Generate prisma client for app...
+Serverless: Remove unused prisma engine:
+Serverless: - node_modules/.prisma/client/libquery_engine-darwin.dylib.node
+Serverless: - node_modules/@prisma/engines/libquery_engine-darwin.dylib.node
+Serverless: - node_modules/@prisma/engines/migration-engine-darwin
+Serverless: - node_modules/@prisma/engines/prisma-fmt-darwin
+Serverless: Copying existing artifacts...
 ```
 
-<Admonition type = "info">
+### Binary targets in <inlinecode>schema.prisma</inlinecode>
 
-The `serverless.yml` configuration file contains the endpoint and function configuration. You can update this file to add or change endpoints. For more AWS specific configuration information, see the AWS provider configuration in the [Serverless Framework documentation](https://serverless.com/framework/docs/providers/aws/guide/functions/).
+The Prisma schema should contain the following in the `generator` block:
 
-</Admonition >
+```prisma
+binaryTargets = ["native", "rhel-openssl-1.0.x"]
+```
 
-## Deploy only the required files
+This is necessary because the local runtime is different from the Lambda runtime. Add the [`binaryTarget`](/reference/api-reference/prisma-schema-reference#binarytargets-options) to make the compatible Prisma engine binary available.
```

Comment:
```suggestion
This is necessary because the local runtime is different from the Lambda runtime. Add the [`binaryTarget`](/reference/api-reference/prisma-schema-reference#binarytargets-options) to make the compatible Prisma engine file available.
```